web/profiles/custom/yalesites_profile/modules/custom/ys_core/src/CoreSettingsManager.php

```diff
@@ -0,0 +1,69 @@
+<?php
```

Comment:
I'm curious about CoreSettingsManager.php. I like having these kinds of services to better organize the business logic. I like encapsulating work that has a single purpose. It's especially important when we need to DRY up code.

But currently this file does not seem to do much. get calls the configs get, and set calls the configs set. So why use this middleware to manage that transaction? It would be different if there was some filter or other business logic taking over, but right now it's a clear pass through.

I'm not against it, but would like to hear your take on it.

Reply:
I can remove this and replace with direct config calls - no problem there. I was following the pattern we established with the `ys_themes` module so if we needed to add any more logic later, we could do it here. But, agreed, we can cross that bridge if we need to later.